GCMessengerSDKSample/app/src/main/java/com/genesys/cloud/messenger/sample/MainActivity.kt

```diff
@@ -150,6 +151,9 @@ class MainActivity : AppCompatActivity(), ChatEventListener {
     }
 
     override fun onBackPressed() {
+
+        dismissChatSnackBar?.takeIf { it.isShown }?.dismiss()
```

Comment:
If a snackbar visible and user press back isn't it expected to dismiss only the snack and on the next back to do the actual back on the open screen